js/network.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+window.network = (function () {
+
+  return {
+    errorHandler: function (message) {
+      console.log('Server error: ', message);
+    },
+
+    successHandler: function (message) {
+      console.log('Server responce: ', message);
```

Comment:
And use templates to show messages to the user. Don't need console.logs

Reply:
done